src/commands/createWebApp/setDefaultRgPlanNameLocation.ts

```diff
@@ -9,55 +9,71 @@ import { AppServicePlan } from "azure-arm-website/lib/models";
 import { MessageItem } from "vscode";
 import { IAppServiceWizardContext, SiteNameStep, WebsiteOS } from "vscode-azureappservice";
 import { createAzureClient, DialogResponses, IActionContext } from "vscode-azureextensionui";
+import { LocationListStep } from 'vscode-azureextensionui';
 import { ext } from "../../extensionVariables";
+import { getResourceGroupFromId } from '../../utils/azureUtils';
 import { nonNullProp } from "../../utils/nonNull";
 import { getWorkspaceSetting, updateGlobalSetting } from "../../vsCodeConfig/settings";
+import { AzConfig, AzConfigProperty, readAzConfig } from "./readAzConfig";
 
 const maxNumberOfSites: number = 3;
 
-export async function setDefaultRgAndPlanName(wizardContext: IAppServiceWizardContext, siteNameStep: SiteNameStep): Promise<void> {
-    // this should always be set when in the basic creation scenario
+export async function setDefaultRgPlanNameLocation(wizardContext: IAppServiceWizardContext, siteNameStep: SiteNameStep): Promise<void> {
+    const config: AzConfig = await readAzConfig(wizardContext, AzConfigProperty.group, AzConfigProperty.location);
```

Comment:
I would add a comment above this line saying something like this:
```
// Reading az config should always happen after prompting because it can cause a few seconds delay
```

Seems like something we could forget